source/libs/api.js

```diff
@@ -2,7 +2,7 @@ import OptionsSync from 'webext-options-sync';
 
 const cache = new Map();
 
-export default async endpoint => {
+export default async (endpoint, options = {logError: true}) => {
```

Comment:
This will only work when there is a single option. Instead, use a normal argument here and use object-spread in side the function.

```js
export default async (endpoint, options) => {
	options = {
		logError: true,
		...options
	};
```